### ISSUE TITLE
[BUG]: Tony Hawk's Downhill Jam: Tutorial Hangs

### ISSUE THREAD

**Issue by leonardokruz** [PCSX2/pcsx2#11629]
![imagem_2024-07-27_002811308](https://github.com/user-attachments/assets/1082ccd0-4b5a-428f-a1d3-f967cd059415)
### Describe the Bug

At the end of the tutorial the game doesn't respond to any button presses and it get stuck, i can't press X to continue or Start to bring the menu up.

### Reproduction Steps

End the tutorial.

### PCSX2 Revision

v2.1.31

### Operating System

Windows 10 (64bit)

### CPU

AMD Ryzen 7 2700

### GPU

GTX 1660

**Comment by Mrlinkwii** [comment]
![image](https://github.com/user-attachments/assets/27aac9c2-6dc3-44cc-8da1-3be9ad559ab8)
cannot replicate here 

have you verified your disc image ?

**Comment by terremoth** [comment]
I will try to reproduce right now.

**Comment by terremoth** [comment]
Yeah, happened with me here too, I chose Tony as my character, the end of the tutorial indeed hangs, I can see him moving a bit his head, is like, the game continues to run, but some kind of bug prevents to continue. Happens just after saving after finish tutorial.

![WhatsApp Image 2024-08-22 at 01 02 13](https://github.com/user-attachments/assets/07e975ad-4d43-46bd-9121-6169d1e95036)

The game CRC is 7D13310A
SLUS-21456 
NTSC-U 

![WhatsApp Image 2024-08-22 at 01 05 30](https://github.com/user-attachments/assets/fdecacb2-ece1-4bc0-8032-8be59725e557)

**Comment by Farran666** [comment]
Savestate: [SLUS-21456 (7D13310A).01.p2s.zip](https://github.com/user-attachments/files/16741509/SLUS-21456.7D13310A.01.p2s.zip)

It can be fixed by `disabling MTVU`, `Instant VU1`, `underclocking the ee`, and `Delay VIF1 Stalls`.

**Comment by terremoth** [comment]
Is there a way to permanently fix this instead of having to manually change these settings? Like a "real" fix?
